### PR TITLE
chore(wren-ai-service): refine langfuse

### PR DESCRIPTION
**Pull request by cyyeh**
- refine langfuse metadata and output

### CODE REVIEW

**Review by paopa — COMMENTED** [review]
overall LGTM. have two minor comments.

File: wren-ai-service/src/utils.py [hunk 1]
```diff
@@ -130,7 +130,7 @@ def init_langfuse():
     host = os.getenv("LANGFUSE_HOST", "https://cloud.langfuse.com")
 
     langfuse_context.configure(
-        enabled=False if enabled.lower() == "false" else True,
+        enabled=(False if enabled.lower() == "false" or not enabled else True),
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/src/utils.py:133]: sure

File: wren-ai-service/src/utils.py [hunk 2]
```diff
@@ -174,18 +174,27 @@ def extract(*args) -> dict:
...
+        langfuse_metadata["mdl_hash"] = metadata.get("mdl_hash")
         langfuse_context.update_current_trace(
             user_id=metadata.get("project_id"),
             session_id=metadata.get("thread_id"),
             release=SERVICE_VERSION,
-            metadata={
-                **MODELS_METADATA,
-                "mdl_hash": metadata.get("mdl_hash"),
-            },
+            metadata=langfuse_metadata,
```
[the remark this answers is not in the document]
Comment (cyyeh) [reply wren-ai-service/src/utils.py]: updated

**Review by paopa — APPROVED** [review]
LGTM. thanks for the patch!

**Pull request merged**